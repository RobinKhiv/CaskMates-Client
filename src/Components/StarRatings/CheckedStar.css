.checked {
    color: rgb(199, 199, 43);
  }