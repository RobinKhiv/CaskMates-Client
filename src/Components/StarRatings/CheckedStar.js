import React, { Component } from 'react'
import './CheckedStar.css'

export default class CheckedStar extends Component {
    handleCheckedStar = value => {
      if(value === undefined)
        return ""
      return value.map(el => <span className="fa fa-star checked"></span>);
    }
    
    render(){
    return (
      <React.Fragment>
        {this.handleCheckedStar(this.props.checkedStar)}
      </React.Fragment>
      )}
  }
  