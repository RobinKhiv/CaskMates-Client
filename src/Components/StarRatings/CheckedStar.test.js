import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import CheckedStar from './CheckedStar';

describe('<CheckedStar />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render( <CheckedStar /> , div);
        ReactDOM.unmountComponentAtNode(div);
    });

  it('renders without crashing', () => {
    const tree = renderer
      .create(<CheckedStar />)
      .toJSON();
    expect(tree).toMatchSnapshot(); 
  });
});