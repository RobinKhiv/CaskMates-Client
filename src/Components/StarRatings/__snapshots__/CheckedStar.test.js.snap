// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckedStar /> renders without crashing 1`] = `""`;
